Makefile

```diff
@@ -810,7 +809,7 @@ venv/bin/ambassador: venv/bin/activate ambassador/requirements.txt
 	@releng/install-py.sh dev install ambassador/requirements.txt
 	@releng/fix_kube_client
 
-venv/bin/activate: dev-requirements.txt multi/requirements.txt kat/requirements.txt
+venv/bin/activate: dev-requirements.txt multi/requirements.txt
```

Comment:
Hm. So I see that you've moved the contents of `kat/requirements` into `ambassador/requirements`, and that's prompting me to think a bit more. 

I think that `kat` should actually probably stay out of the `ambassador` directory, because `ambassador` is the stuff we ship as part of the delivered image. Delivering `kat` to an end user, rather than a developer, is probably unnecessary.